src/ert/_c_wrappers/enkf/enkf_fs.py

```diff
@@ -234,6 +232,46 @@ def load_parameter(
             self, ensemble_config, iens_active_index, parameter
         )
 
+    def save_field_data(
+        self,
+        parameter_name: str,
+        realization: int,
+        data: npt.ArrayLike,
+    ) -> None:
+        output_path = self.mount_point / f"field-{realization}"
+        Path.mkdir(output_path, exist_ok=True)
+
+        # get base grid file (save?)
+        # set data1d
+        np.save(f"{output_path}/{parameter_name}", data)
+
+        self.getStateMap().update_matching(
```

Comment:
Should this happen here? Maybe it should happen one level higher in the callstack? Or ideally we should add a single enum per parameter, but that sounds like it is outside the scope of this PR, and should be an issue instead.

Reply:
We are updating the statemap several other places in enk_fs.py. Perhaps we can leave it as is and create a new PR looking into refactoring this

Reply:
Sounds good, could you create an issue for that?